csrankings-z.csv

```diff
@@ -47,7 +47,7 @@ Zeyu Wang 0003,HKUST,https://zachzeyuwang.github.io/,q7NLPG0AAAAJ
 Zeyun Yu,University of Wisconsin - Milwaukee,https://uwm.edu/engineering/people/yu-ph-d-zeyun,9kT9SaIAAAAJ
 Zhan Qin,Zhejiang University,https://person.zju.edu.cn/qinzhan,5fa4lOQAAAAJ
 Zhang Xiong 0001,Beihang University,http://scse.buaa.edu.cn/info/1078/2637.htm,NOSCHOLARPAGE
-Zhangyang Wang,Texas A&M University,http://www.atlaswang.com,pxFyKAIAAAAJ
+Zhangyang Wang,The University of Texas at Austin,https://vita-group.github.io,pxFyKAIAAAAJ

```

Comment:
The institution name in the database is "University of Texas at Austin"; please fix and I will merge.

Reply:
Please address this issue ASAP; I am closing old PRs by the end of this week.

Reply:
Hi, just commit with the correct name. Thanks!